<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBookAir.fios-router.home" failures="1" tests="32" name="testRunner.testRunner" time="9.232" errors="0" timestamp="2025-02-07T22:34:57 EST" skipped="0">
  <testcase classname="testRunner.testRunner" name="runScenario" time="3.257"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.198"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.174"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.186"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.217"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.095"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.201"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.202"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.192"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.591"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.099"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.177"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.184"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.200"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.214"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.091"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.180"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.179"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.238"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.200"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.195"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.192">
    <failure message="expected [201] but found [500]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [201] but found [500]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at stepDefinitions.createUserStepDefinition.admin_receives_status_for_create_user(createUserStepDefinition.java:93)
at âœ½.Admin receives "201" "Created" Status for create user(file:///Users/mathumathibalakrishnan/eclipse/Mathumathi_2024/2025_Team04_RestAssured_BDD/src/test/resources/features/02_createUser.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.187"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.101"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.186"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.202"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.187"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.232"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.193"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.101"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.188"/>
  <testcase classname="testRunner.testRunner" name="runScenario" time="0.193"/>
</testsuite> <!-- testRunner.testRunner -->
